### PR TITLE
(maint) Test correct branches for 5.1.x and 2.x

### PR DESCRIPTION
**Pull request by haus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ notifications:
 branches:
   only:
     - master
+    - 5.1.x
     - stable
```
Comment (Magisus) [inline .travis.yml]: Can we get rid of stable here?

**Comment by Magisus** [comment]
I'm actually kind of curious why we're even restricting the branches Travis runs on at all. I don't think most of our other projects do this, and I'm not convinced it's buying us much, especially given the new branching scheme that uses version numbers.

**Comment by haus** [comment]
Yea i'm not sure. If i remove the branches will it just run everywhere?

**Comment by Magisus** [comment]
That's what it looks like. The Puppet repo does not have a `branches` section.

**Comment by haus** [comment]
This is failing travis for the thing @justinstoller 's PR fixes

**Comment by Magisus** [comment]
Should this be merged into 2.x?

**Comment by haus** [comment]
@Magisus no idea. i can retarget there and we can just merge up all the way

**Comment by haus** [comment]
or whatever

**Comment by haus** [comment]
so good

**Comment by haus** [comment]
replaced by https://github.com/puppetlabs/puppetserver/pull/1591

**Pull request closed without merging**